src/OSPSuite.InstallationValidator/Views/MainView.cs

```diff
@@ -75,5 +68,20 @@ public void ValidationIsRunning(bool validationRunning)
       {
          OkEnabled = !HasError && !validationRunning;
       }
+
+      public void AppendText(string newText)
+      {
+         getInnerTextBox(memoEditLogMessages).AppendText(newText);
+      }
+
+      private TextBox getInnerTextBox(DevExpress.XtraEditors.TextEdit editor)
+      {
+         return editor?.Controls.OfType<TextBox>().FirstOrDefault();
```

Comment:
Nice. Did not know about that. Is it work an extension method in OSPSuite.UI for MemoEdit?